clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -240,10 +238,11 @@ void BackendConsumer::HandleTranslationUnit(ASTContext &C) {
 
     Gen->HandleTranslationUnit(C);
 
-    if (TimerIsEnabled && !--LLVMIRGenerationRefCount)
-      LLVMIRGeneration.yieldTo(CI.getFrontendTimer());
-
-    IRGenFinished = true;
```

Comment:
Why we deleted `IRGenFinished = true;` here?

Reply:
Ah, must've been a bad merge.

Reply:
Good point.